BDArmory/Weapons/Missiles/BDModularGuidance.cs

```diff
@@ -1159,7 +1159,7 @@ public override void Detonate()
                 if (explosiveParts != null)
                 {
                     foreach (var explosivePart in explosiveParts)
-                    { if (!explosivePart.manualOverride) explosivePart.DetonateIfPossible(); }
+                    { if (!explosivePart.manualOverride) explosivePart.DetonateIfPossible(GetForwardTransform()); }
```

Comment:
This is the forward transform of the missile, which isn't necessarily the same as the orientation of the warhead.